task/buildah-min/0.4/buildah-min.yaml

```diff
@@ -481,7 +481,19 @@ spec:
         if ! grep -E "^[^#]*subscription-manager.[^#]*register" "$dockerfile_path"; then
           # user is not running registration in the Containerfile: pre-register.
           echo "Pre-registering with subscription manager."
-          subscription-manager register --org "$(cat /tmp/activation-key/org)" --activationkey "$(cat /tmp/activation-key/activationkey)"
+          for i in {1..5}; do
```

Comment:
@Victoremepunto Can you please provide some context for this change in the commit message? Obviously we're failing some kind of rejection, but that's not enough of a justification for future reference and git history bookkeeping, IOW I've never faced a subman registration issue, so apparently this is caused by automation AND/OR environment and timing, can you please add more specifics on what led to this particular change?

Reply:
Splendid! So based on your description, please mention something along those lines in the commit message without linking to private resources (**only public ones** if existing, or just direct copy-paste from the error logs you've seen), that will be enough context to accompany the change :).